cheroot/cli.py

```diff
@@ -49,6 +49,7 @@ def __init__(self, address, port):
         Args:
             address (str): Host name or IP address
             port (int): TCP port number
+
```

Comment:
Why is this necessary?

Reply:
I went full _follow the linter_, and that missing blank line caused error `D413` from this list: http://www.pydocstyle.org/en/latest/error_codes.html

Reply:
Interesting. I guess we'll have to switch this to proper sphinx-recognized params anyway. But that's later.